samples/client/petstore/python-asyncio/petstore_api/models/format_test.py

```diff
@@ -321,27 +321,27 @@ def binary(self, binary):
         self._binary = binary
 
     @property
-    def date(self):
-        """Gets the date of this FormatTest.  # noqa: E501
+    def _date(self):
+        """Gets the _date of this FormatTest.  # noqa: E501
 
 
-        :return: The date of this FormatTest.  # noqa: E501
+        :return: The _date of this FormatTest.  # noqa: E501
         :rtype: date
         """
-        return self._date
+        return self.__date
 
-    @date.setter
-    def date(self, date):
-        """Sets the date of this FormatTest.
+    @_date.setter
+    def _date(self, _date):
+        """Sets the _date of this FormatTest.
 
 
-        :param date: The date of this FormatTest.  # noqa: E501
+        :param _date: The _date of this FormatTest.  # noqa: E501
         :type: date
         """
-        if date is None:
-            raise ValueError("Invalid value for `date`, must not be `None`")  # noqa: E501
+        if _date is None:
+            raise ValueError("Invalid value for `_date`, must not be `None`")  # noqa: E501
 
-        self._date = date
+        self.__date = _date
```

Comment:
I don't think the double underscore is really necessary here (nor anywhere else).

https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles
`__double_leading_underscore: when naming a class attribute, invokes name mangling (inside class FooBar, __boo becomes _FooBar__boo; see below).`